src/components/Score/Score.stories.tsx

```diff
@@ -36,7 +36,9 @@ export const Table: StoryObj<Args> = {
   decorators: [
     (Story) => (
       <div>
-        <p>Score has transparent border for use in Table component</p>
+        <p className="mb-4">
```

Comment:
`<p>` => `Text` ?

Reply:
Yea we can use the `Text` component here with the same TW class but since the purpose of this story is to display the `<Score>` component I don't think it's a big deal which we use